fluent/client/ws/connection.go

```diff
@@ -58,6 +65,8 @@ type Connection interface {
 
 type connection struct {
 	ext.Conn
+	log           bool
+	logger        Logger
```

Comment:
These will create conflicting props - (using two vars to track the same state, which can be done using a single var only ).  By just using the logger != nil. We do not need to maintain an internal state for it.

Reply:
fixed